### ISSUE TITLE
Make tooltip buttons type="button"

### ISSUE THREAD

**Issue by acharb** [stellar/stellar-design-system#115]
If the tooltip is in a form currently it submits the form on click

**Comment by quietbits** [comment]
@acharb Could you please provide a code example of where this issue is happening? I'm assuming it's on forms that have `onSubmit` on the form, correct?

**Comment by acharb** [comment]
yup! For example here: https://github.com/stellar/freighter/pull/432/files#diff-40870d652ae0f03471886bc3aead9e4e9ad2a3440905e85d881f79aa1b90d283R106-R118
(RadioCheck contains a DetailsTooltip, which submits the form)

**Comment by piyalbasu** [comment]
I believe if you just give the button `type=button` it will prevent it from submitting

**Comment by quietbits** [comment]
Thanks for the details. I just wanted to confirm that I understood the issue correctly. It looks like browsers usually have the default type for buttons `submit`, that's why this is happening.
